jscomp/outcome_printer/tweaked_reason_oprint.cppo.ml

```diff
@@ -152,7 +152,7 @@ let print_out_value ppf tree =
 #if OCAML_VERSION >= (4,3,0) && not defined BS_NO_COMPILER_PATCH
     | Oval_string (s,_,_) ->
 #else
-    | Oval_string s ->
+    | Oval_string (s,_,_) ->
```

Comment:
curious why this change is needed to make 4.06 build, should not it go from the first `#if`

Reply:
I'm not sure. My workflow is a little different from the recommended in CONTIBUTING.md - it's something like:

```sh
git pull
cd ocaml
git checkout 4.06.1+BS && git pull
cd ..
ninja -C jscomp -t clean
ninja -C jscomp # this command fails if I don't change to the three
yarn postinstall
```